src/components/SettingDialog.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <q-dialog
+    maximized
+    persistent
+    class="setting-dialog"
+    v-model="modelValueComputed"
+    transition-show="slide-up"
+    transition-hide="slide-down"
+  >
+    <q-layout container view="hhh lpr fFf" class="bg-white">
+      <q-footer reveal elevated>
+        <q-toolbar>
+          <q-toolbar-title class="text-secondary">オプション</q-toolbar-title>
+        </q-toolbar>
+      </q-footer>
```

Comment:
In the home window or help window, there is a banner at the top.
How about a header to prevent confusion for the user?